docs/asciidoc/http-server.adoc

```diff
@@ -366,6 +365,48 @@ You can use the following configuration (for Logback or similar logging framewor
 ----
 ====
 
+The following example enables it programmatically:
+====
+[source,java,indent=0]
+.{examplesdir}/accessLog/Application.java
+----
+include::{examplesdir}/accessLog/Application.java[lines=18..32]
+----
+====
+
+Calling this method will take precedence over the system property configuration.
+
+By default, the logging format is https://en.wikipedia.org/wiki/Common_Log_Format[Common Log Format], but you can
+specify a custom one as parameter like in the following example:
```

Comment:
Change the line to:

specify a custom one as a parameter, as in the following example: